docs/developer/issue-triage-process.md

```diff
@@ -0,0 +1,90 @@
+# Kubeapps triage process
+
+## Kubeapps backlog
+
+Kubeapps keeps a backlog of issues on GitHub submitted both by maintainers and contributors: this backlog comprises bugs, feature requests, and technical debt. If it’s a bug, an idea for a new feature, or something in between, it’s filed as an issue in the [kubeapps issue page](https://github.com/kubeapps/kubeapps/issues) on GitHub.
+There are some special considerations about how the Kubeapps maintainer team manages its backlog on GitHub:
+- The issue repository is completely open. The maintainer team, along with the entire Kubeapps community, files all feature enhancements, bugs, and potential future work into the open repository.
+- Issues are closed if solved, or outdated (meaning the issue does not apply anymore according to the evolution of Kubeapps or it was waiting for more information which was never received).
+
+## Kubeapps' triage process
+
+The [kubeapps/kubeapps](https://github.com/kubeapps/kubeapps) issues are triaged by any member of the maintainer team. The triage process is defined with the objective of helping the Kubeapps team during the planning of each iteration so that the selected issues provide the highest added value to the product itself according to the Kubeapps maintainer team capacities.
+
+The triage process should be performed manually for any issue in the Inbox column and will consist on:
+- Reading the description of the issue. 
+  * If more information is requested, as a result of the triage process the issue must be labeled as `<awaiting-more-evidence>`, a comment requesting for information added and the issue must remain in the Inbox column.
+- Adding labels for each category (required): component, kind, size and priority.
+- Check if it is a `good-first-issue` to start contributing to Kubeapps and label the issue in consequence.
+- Moving the issue to the appropriate column according within the triage process:
+  - **Committed** → Issues labeled as `priority/unbreak-now`
+  - **Next** iteration discussion → Issues labeled as `priority/high`
+  - **Inbox** → Issues labeled `priority/medium` and `priority/low` will remain in the inbox before moving to the **Backlog** to double check with the whole maintainer team during the iteration planning meeting.
+  - **Inbox** → Issues labeled as `awaiting-more-evidence`
+- Repeat until all issues in the **Inbox** column are triaged.
+
+Besides this manual triage process, there are two automatic process lead by the bots configured in GitHub:
++ Automatically move new and reopened issues to the **Inbox** column to start the triage process (due to eventual project-bot outages, once a month we should manually search for issues with "no:project" assigned).
++ Automatically **dependabot** and **Snyk** are labelling PRs depending on the target language.
```

Comment:
Question: do we prefer en_gb rather than en_us ? I guess so, but perhaps we should check the corporate guidelines to shed some light on it.